training/4-train-lstm.py

```diff
@@ -47,8 +51,14 @@ def prepare_file(file_i, filename, classifiers, f_nick, model_type):
     x = np.array(x)
     x = np.transpose(x)
     if model_type == 'lstm' or model_type == 'gru':
-        x = x.reshape((len(x), 1, len(x[0])))    
-    return x
+        x = x.reshape((len(x), 1, len(x[0])))
+    # Collect the true lables
+    seizures = nedc.loadTSE(os.path.join(EDF_ROOT,filename[:-4]+'.tse'))
+    
+    # Create labels at 1Hz sampling rate
```

Comment:
Comment seems misleading as y will be labels at fs sampling rate

Reply:
Fixed: d845aa7b50db125860ab9e794f8f23d927711717